js/iosrtc.js

```diff
@@ -182,16 +182,15 @@ function registerGlobals(doNotRestoreCallbacksSupport) {
 		navigator.mediaDevices = {};
 	}
 
-	navigator.getUserMedia                  = getUserMedia;
-	navigator.webkitGetUserMedia            = getUserMedia;
-	navigator.mediaDevices.getUserMedia     = getUserMedia;
-	navigator.mediaDevices.enumerateDevices = enumerateDevices;
-
 	// Restore Callback support
 	if (!doNotRestoreCallbacksSupport) {
 		restoreCallbacksSupport();
 	}
 
+	navigator.getUserMedia                  = getUserMedia;
```

Comment:
@hthetiot Had to make this change to start using getUserMedia in our application. lease let me know if I have missed anything else.